meetings/continuity.html

```diff
@@ -170,6 +170,10 @@ <h2 id='virtualPresence'>Arrangements for Virtual Presence</h2>
 	    However, any discussion content that is shared via video, including
 	    screen sharing, must be made <a href='https://www.w3.org/WAI/WCAG21/quickref/'>accessible</a> by some means to those who cannot
 	    -- or choose not to -- use the video feed.</p>
+      <p id='screen-sharing'><span class='subtopic'>Screen sharing</span>
+            While screen sharing can be an effective way to work together, it also
+            causes accessibility problems since it prohibits the use of screen readers and translation services.
+            A good pratice is to share links to material instead of just showing a picture of it.</p>
```

Comment:
```suggestion
            A best practice is to post accessible materials in the web and share links instead of just showing a picture of it.</p>
```